packages/editor/src/client/routes/pageEditor.tsx

```diff
@@ -228,12 +235,23 @@ export function PageEditor({id}: PageEditorProps) {
     }
 
     setChanged(false)
-    Alert.success(t('pageEditor.overview.pagePublished'), 2000)
+    publishDate <= new Date()
+      ? Notification.success({
+          title: t('pageEditor.overview.pagePublished'),
+          duration: 2000
+        })
+      : Notification.success({
+          title: t('pageEditor.overview.pagePending'),
+          duration: 2000
+        })
   }
 
   useEffect(() => {
     if (isNotFound) {
-      Alert.error(t('pageEditor.overview.pageNotFound'), 0)
+      Notification.error({
+        title: t('pageEditor.overview.pageNotFound'),
+        duration: 0
+      })
```

Comment:
`Duration` set to `0` here - is that on purpose ?

Reply:
this should also be an `Alert` and the idea is that error alerts should be visible until the user clicks them away